docs/using.rst

```diff
@@ -44,32 +44,28 @@ a combination_ of distinct authenticator and installer plugins.
 =========== ==== ==== =============================================================== =============================
 Plugin      Auth Inst Notes                                                           Challenge types (and port)
 =========== ==== ==== =============================================================== =============================
-apache_     Y    Y    | Automates obtaining and installing a certificate with Apache  tls-sni-01_ (443)
```

Comment:
Same comment as above -- Nginx and Apache should be modified to talk about HTTP-01 instead.